src/main/java/com/facebook/openwifirrm/Utils.java

```diff
@@ -57,6 +59,40 @@ protected boolean removeEldestEntry(final Map.Entry<K, V> eldest) {
 		}
 	}
 
+	/** Thread factory based on DefaultThreadFactory supporting name prefix. */
+	public static class NamedThreadFactory implements ThreadFactory {
+		private static final AtomicInteger poolNumber = new AtomicInteger(1);
```

Comment:
Maybe add javadocs to all these member vars

Reply:
This was just copy/pasted from the JDK sources, which is why I didn't bother.